torch/onnx/symbolic.py

```diff
@@ -924,6 +924,23 @@ def le(g, input, other):
     return gt_impl(g, input, _if_scalar_type_as(g, other, input))
 
 
+def wrap_logical_op_with_cast_to_and_from_bool(func):
```

Comment:
can we refactor this function with `wrap_logical_op_with_cast_to_uint8`?
so we will have `wrap_func_with_cast_to(func, to_type)`, and you can directly pass type as a parameter.

Reply:
done. I have renamed the other to `inject_logical_op_with_cast_to`. A little context is that in ONNX, `And` and `Or` accept only `Bool` type, so inputs need to be cast to `Bool`, and output cast back to original type.